src/test/java/reposense/parser/RepoConfigParserTest.java

```diff
@@ -35,6 +35,9 @@
             "RepoConfigParserTest/require_trailing_whitespaces/repoconfig_redundantLines_test.csv");
     private static final Path REPO_CONFIG_INVALID_LOCATION_FILE = loadResource(RepoConfigParserTest.class,
             "RepoConfigParserTest/repoconfig_invalidLocation_test.csv");
+    private static final Path REPO_CONFIG_UNRECOGNIZED_VALUES_FOR_YES_KEYWORD_HEADERS_FILE =
+            loadResource(RepoConfigParserTest.class,
+                    "RepoConfigParserTest/repoconfig_unrecognizedValuesForYesKeywordHeaders_test.csv");
```

Comment:
The extra spaces are not necessary.

Reply:
I have yet to make a PR for this branch, I'm not sure which branch to direct the PR to since the branch is non existent on the remote repository. Aside from that I need the commit hashes in my the branch above to be preserved (as it is used in the tests), is there any way you can ensure that?